src/components/molecules/HeatMap.tsx

```diff
@@ -1,9 +1,11 @@
-import 'leaflet-css';
+import 'leaflet/dist/leaflet.css';
```

Comment:
`import 'leaflet/dist/leaflet.css' - its is a simple `css`, which should be imported in one place only
Please open a ticket to include it in app.tsx and remove from all other places